src/ui/libs/DatalensChartkit/modules/axios/axios.ts

```diff
@@ -2,6 +2,7 @@ import type {AxiosError} from 'axios';
 import axios from 'axios';
 import axiosRetry, {isRetryableError} from 'axios-retry';
 import isNumber from 'lodash/isNumber';
+import {sleep} from 'shared';
```

Comment:
Lets grab this helper like `shared/modules` to avoid global indexe problem

Reply:
agree, fixed